src/quip_entity/message.py

```diff
@@ -18,11 +18,7 @@ def __init__(self, text, author_id, author_name, created_usec, updated_usec):
         self.edited = updated_usec - created_usec >= 10**6
         self.timestamp = datetime.fromtimestamp(max(updated_usec, created_usec)/(10**6))
 
-    def to_json(self):
-        return {
-            'text': self.text,
-            'author_id': self.author_id,
-            'author_name': self.author_name,
-            'edited': self.edited,
-            'timestamp': self.timestamp.strftime('%Y-%m-%d %H-%M-%S')
-        }
+    def __str__(self):
+        return "%s | %s [%s] %s: %s\n" % (self.author_id, self.author_name, 
```

Comment:
Let's remove new line '\n' here and will use it in join methods (e.g. '\n'.join(messages))